dcc-portal-ui/app/scripts/app/js/app.js

```diff
@@ -488,10 +488,13 @@
       gettextCatalog.setCurrentLanguage('en_CA');
 
       HistoryManager.addToIgnoreScrollResetWhiteList(['analysis','advanced', 'compound']);
-    
-      // Add UI Router Debug if there is a fatal state change error
-      $rootScope.$on('$stateChangeError', function () {
-        console.error('State Change Error Occurred. Error occurred with arguments: ', arguments);
+      
+      $rootScope.$on('$stateChangeError', function(event, toState, toParams) {
+        $state.go('404', {page: toState.name, id: toParams.id, url: toState.url}, {location: false});
```

Comment:
No, 404 means "not found". Js errors are not the same as they are not the user's fault